build-tools/create-pkg/create-pkg.targets

```diff
@@ -0,0 +1,92 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\installers\create-installers.targets" />
+  <PropertyGroup>
+    <PkgInstallDir>/</PkgInstallDir>
+    <PkgInstallDir Condition="$([MSBuild]::IsOsPlatform(Linux))">/usr</PkgInstallDir>
+    <PayloadDir>$(OutputPath)/pkgarchive</PayloadDir>
+    <PkgOutputPath>$(OutputPath)/packages</PkgOutputPath>
+  </PropertyGroup>
+  <Target Name="_CopyFilesToPayloadDir"
+      DependsOnTargets="GetXAVersion" >
+    <PropertyGroup>
+      <XAFrameworkDir>$(PayloadDir)/Library/Frameworks/Xamarin.Android.framework/Versions/$(XAVersion)</XAFrameworkDir>
+      <MonoFrameworkExternalDir>$(PayloadDir)/Library/Frameworks/Mono.framework/External</MonoFrameworkExternalDir>
+      <XBuildDir>$(XAFrameworkDir)/lib/xamarin.android/xbuild/Xamarin/Android</XBuildDir>
+      <XBuildFrameworkDir>$(XAFrameworkDir)/lib/xamarin.android/xbuild-frameworks/MonoAndroid</XBuildFrameworkDir>
+    </PropertyGroup>
+    <RemoveDir Directories="$(PayloadDir)" />
+    <Copy
+        SourceFiles="@(FrameworkFiles)"
+        DestinationFolder="$(XBuildFrameworkDir)/%(RecursiveDir)"
+    />
+    <Copy
+        SourceFiles="@(FirstFrameworkFiles)"
+        DestinationFolder="$(XBuildFrameworkDir)/$(FirstFrameworkVersion)/%(RecursiveDir)"
+    />
+    <Copy
+        SourceFiles="@(BCLFrameworkFiles)"
+        DestinationFolder="$(XBuildFrameworkDir)/$(BclFrameworkVersion)/%(RecursiveDir)"
+    />
+    <Copy
+        SourceFiles="@(MSBuildFiles);@(MSBuildFilesUnix)"
+        DestinationFolder="$(XBuildDir)/%(RecursiveDir)"
+    />
+    <Copy
+        SourceFiles="@(MSBuildFilesDarwin)"
+        DestinationFolder="$(XBuildDir)/Darwin"
+    />
+    <Copy
+        SourceFiles="@(MSBuildLibFiles)"
+        DestinationFolder="$(XBuildDir)/lib/%(RecursiveDir)"
+    />
+    <Copy
+        SourceFiles="@(MSBuildLibFilesDarwinHost)"
+        DestinationFolder="$(XBuildDir)/lib/host-Darwin"
+    />
+    <Copy
+        SourceFiles="@(VersionFiles)"
+        DestinationFolder="$(XAFrameworkDir)"
+    />
+    <Copy
+        SourceFiles="@(TargetsFiles)"
+        DestinationFolder="$(XAFrameworkDir)/lib/xamarin.android/xbuild/%(RecursiveDir)"
+    />
+  </Target>
+  <Target Name="_CreateSymbolicLinks"
+      DependsOnTargets="_CopyFilesToPayloadDir">
```

Comment:
Just like the `<Copy/>` calls above, these `<Exec/>` would always run on incremental builds.

I would maybe add a `Condition="!Exists('path\to\foo')"`, so the `<Exec/>` calls can skip.